packages/design-system/src/components/EmptyState/variants/EmptyStateMedium.module.scss

```diff
@@ -0,0 +1,6 @@
+@use '~@talend/design-tokens/lib/tokens';
+
+.mediumIllustration {
+	width: 4rem;
```

Comment:
either you import and use tokens (please, please, please) or you remove the import

Reply:
Sure. 

In this case, I don't care about tokens sizes. Our current token grid for size isn't build for that, it's very much centered around paddings, margins, and form element sizes (inputs and buttons). 

I'm open to having a different set of tokens for UI sizes, but in the meantime, illustrations have very little reason to stick to those tokens - especially if they keep to a base 4 ruleset still.